File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/LocalDbMboService.java
package com.lenta.lentabp10.MobileObjectsService;

import com.lenta.lentabp10.MobileObjectsService.models.MB_S_07;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_14;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_22;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_25;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_26;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_30;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_31;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_32;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_33;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_34;
import com.lenta.lentabp10.MobileObjectsService.models.MB_S_36;

import java.util.List;

public class LocalDbMboService implements IMboService{
    @Override
    public MB_S_07 getUomInfo(String uom) {
        return null;
    }

    @Override
    public List<MB_S_14> getParametersByParamName(String paramName) {
        return null;
    }

    @Override
    public String getFirstParameterValueByName(String paramName) {
        return null;
    }

    @Override
    public List<MB_S_22> getSetItemsByMatnrOsn(String matnrOsn) {
        return null;
    }

    @Override
    public MB_S_25 getBarcodeInfo(String barcode) {
        return null;
    }

    @Override
    public List<MB_S_26> getAllPrinters() {
        return null;
    }

    @Override
    public MB_S_30 getProductByMaterialNumber(String material) {
        return null;
    }

    @Override
    public MB_S_31 getWriteOffCause(String taskType, String sectionId, String matkl, String ekgrp) {
        return null;
    }

    @Override
    public List<MB_S_32> getWriteOffCauseByTask(String taskType) {
        return null;
    }

    @Override
    public List<MB_S_33> getStoragesByTaskTypeAndTK(String taskType, String tkNumber) {
        return null;
    }

    @Override
    public List<MB_S_34> getProductTypesByTaskType(String taskType) {
        return null;
    }

    @Override
    public MB_S_36 getGisControlsByTaskCntrl(String taskCntrl) {
        return null;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/LocalDbMboService.kt
package com.lenta.bp10.mobileObjectsService

import com.lenta.bp10.mobileObjectsService.models.*

class LocalDbMboService : IMboService {
    override fun getUomInfo(uom: String): MB_S_07 {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
        /**val uomInfo = PR_WOB.MB_S_07.FindByPrimaryKey(uom)

        return if (uomInfo == null)
            null
        else
            MB_S_07(uomInfo!!.UOM, uomInfo!!.NAME, uomInfo!!.DECAN)*/
    }

    override fun getParametersByParamName(paramName: String): List<MB_S_14> {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getFirstParameterValueByName(paramName: String): String {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getSetItemsByMatnrOsn(matnrOsn: String): List<MB_S_22> {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getBarcodeInfo(barcode: String): MB_S_25 {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getAllPrinters(): List<MB_S_26> {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getProductByMaterialNumber(material: String): MB_S_30 {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getWriteOffCause(taskType: String, sectionId: String, matkl: String, ekgrp: String): MB_S_31 {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getWriteOffCauseByTask(taskType: String): List<MB_S_32> {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getStoragesByTaskTypeAndTK(taskType: String, tkNumber: String): List<MB_S_33> {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getProductTypesByTaskType(taskType: String): List<MB_S_34> {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }

    override fun getGisControlsByTaskCntrl(taskCntrl: String): MB_S_36 {
        TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
    }
}